CODEOWNERS

```diff
@@ -1,8 +1,78 @@
-# Beatriz Rizental as main contact for addon related changes
-/addons/ @brizental
-/src/addons/ @brizental
-/signature/ @brizental
+# These owners will be the default owners for each directory
+# in the repo.
 
-# Francesco Lodolo as main contact for string changes
+/addons/ @brizental @bakulf @MattLichtenstein
 addons/**/manifest.json @flodolo
-translations/strings.yaml @flodolo
+
+/3rdparty/adjust-android-sdk/ @bakulf @birdsarah
+/3rdparty/adjust-ios-sdk/ @bakulf @birdsarah
+/3rdparty/glean/ @brizental
+/3rdparty/openSSL/ @strseb
+/3rdparty/sentry/ @strseb
+/3rdparty/wireguard-apple/ @bakulf @birdsarah @mcleinman
+/3rdparty/wireguard-go/ @bakulf @oskirby
+/3rdparty/wireguard-tools/ @bakulf @oskirby
+
+/android/ @strseb
+/balrog/ @bakulf @birdsarah
+/ci_scripts/ @birdsarah
+/docs/ @birdsarah
```

Comment:
I would remove this entry otherwise Sarah has to review all the changes in docs...